docs/src/app/pages/BluetoothLEIntro.md

```diff
@@ -53,226 +89,740 @@ Bluetooth Low Energy, also referred to as Bluetooth LE or simply BLE, is a new p
+       A list of values to write to the device.
 
 ![call BluetoothLE1 WriteBytesserviceUuidcharacteristicUuidsignedvalues](blocks/BluetoothLE.WriteBytes.svg)
 
-+ <a name="WriteBytesWithResponse"></a>`WriteBytesWithResponse` – Writes one or more 8-bit integer values to a connected BluetoothLE device and wait for an acknowledgement via the ByteValuesWritten event. Service Unique ID and Characteristic Unique ID are required. The values parameter can either be a single numeric value or a list of values. If signed is true, the acceptable values are between -128 and 127. If signed is false, the acceptable values are between 0 and 255.
++ <a name="WriteBytesWithResponse"></a>`WriteBytesWithResponse` – Writes one or more 8-bit integer values to a connected BluetoothLE device and waits for an
+ acknowledgement via the <a href="#BytesWritten"><code>BytesWritten</code></a> event.
+ Service Unique ID and Characteristic Unique ID are required. The values parameter can either
+ be a single numeric value or a list of values. If signed is true, the acceptable values are
+ between -128 and 127. If signed is false, the acceptable values are between 0 and 255.
+
+ __Parameters__:
+
+     * <code>serviceUuid</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/text.html#string">_text_</a>) &mdash;
+       The unique identifier of the service passed in the read or register call.
+     * <code>characteristicUuid</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/text.html#string">_text_</a>) &mdash;
+       The unique identifier of the characteristic in the read or register call.
+     * <code>signed</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/logic.html#true" target="_blank">_boolean_</a>)
+       Interpret the bytes as signed (true) or unsigned (false).
+     * <code>values</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/lists.html#makealist">_list_</a>) &mdash;
+       A list of values to write to the device.
 
 ![call BluetoothLE1 WriteBytesWithResponseserviceUuidcharacteristicUuidsignedvalues](blocks/BluetoothLE.WriteBytesWithResponse.svg)
 
-+ <a name="WriteFloats"></a>`WriteFloats` – Writes one or more floating point values to a connected BluetoothLE device. Service Unique ID and Characteristic Unique ID are required. The values parameter can either be a single numeric value or a list of values. If shortFloat is true, the values will be compressed to 16-bits. If shortFloat is false, the values are sent as 32-bit.
++ <a name="WriteFloats"></a>`WriteFloats` – Writes one or more IEEE 754 floating point numbers to a connected BluetoothLE device. Service Unique ID
+ and Characteristic Unique ID are required. The values parameter can either be a single numeric
+ value or a list of values. If <code>shortFloats</code> is true, then each numeric value will be
```

Comment:
Block image says "shortFloat" without the 's'. The description says "shortFloats" three times